### ISSUE TITLE
Jewelcrafting stuck at 535 on 

### ISSUE THREAD

**Issue by chaoslord** [Valhalla-Gaming/Tracker#4442]
hello, first of all thank you for your time and pardon my english..
the problem is on pg named Sabbadon:
I trained jewelcrafting zen master (600) as i'm draenei i can go up to 610.
i farmed materials and started to upgrade but when i arrived to 535, i remained stuck there.
All the recipes i tried were "orange" but got no points.
I thought maybe the recipes where broken, but i tried many.
disconnected-cleared cache- and all standard stuffs. problem remained.
talking with Buzan i thought that maybe the problem is not broken recipes but broken trainer, as i'm stuck at maximum Cataclysm level for a draenei (525+10).
on my profession tab i see Jewelcrafting 535/610.
I bought the "zen master" from jewelcrafter trainer located in Stormwind.
Examples of recipes i tried while chatting with Buzan are:
Mysterious Roguestone - Fractured Sunstone - Delicate Pandarian Garnet - Nimble Alexandrite

After PM to Tregeta and Max, Tregeta told me to place this on the issue tracker

**Comment by Trekex** [comment]
I trained JC some time ago and I have it on 600 atm, so maybe is some recent bug

**Comment by lavalash551** [comment]
It works perfect. You have to look for NPC http://www.wowhead.com/npc=65098/mai-the-jade-shaper to train new reciples.

**Comment by chaoslord** [comment]
@Trekex: yes, issue seems to be new: Tregeta asked me to add it to traker because need to check if bug on my pg or not. maybe it worked before :) 
@lavalash551: the npc you posted is not working here..

**Comment by Tregeta** [comment]
@Chaoslord can you explain how this npc is not working for you?

**Comment by chaoslord** [comment]
@Tregeta: well, as I wrote, i trained Zen Master Jewelcrafting in Stormwind and i bought all recipes I could use and started to create items. at 535 I found that all "orange" recipes I was creating no points were added, always stuck at 535.
Tried many recipes, same result. i look in my profession book and see 535/610 so I really bought the upgrade to Zen Master, but seems like i'm stuck. as i'm draenei i have +10 in jewelcrafting, seems like i'm stuck at end maximum level reachable on Cataclysm, like the Zen Master Trainer is bugged on my pg then i tried other 3 recipes (all orange and in that order) few mins ago and found that Shifting Roguestone and Solid Lapis Lazuli gave me 0 points, but when i tried Vermillion Onyx i received correctly 3 points, Delicate Pandarian Garnet and gave 0 points! 
So the problem is that most of orange recipes don't give points on creating items.
So definitely the problems

**Comment by Tregeta** [comment]
I was pertaining to the NPC, I already have the data from that as it is in your original post.

**Comment by chaoslord** [comment]
@Tregeta good, hope i was helpful somehow :)
